libcst/codemod/commands/remove_unused_imports.py

```diff
@@ -17,21 +26,60 @@ class RemoveUnusedImportsCommand(VisitorBasedCodemodCommand):
     to track cross-references between them. If a symbol is imported in a file
     but otherwise unused in it, that import will be removed even if it is being
     referenced from another file.
-
-    It currently doesn't keep track of string type annotations, so an import
-    for `MyType` used only in `def f() -> "MyType"` will be removed.
     """
 
     DESCRIPTION: str = (
         "Remove all imports that are not used in a file. "
         "Note: only considers the file in isolation. "
-        "Note: does not account for usages in string type annotations. "
     )
 
+    METADATA_DEPENDENCIES = (
+        PositionProvider,
+        *GatherCommentsVisitor.METADATA_DEPENDENCIES,
```

Comment:
We should only need to list the direct metadata dependency of `RemoveUnusedImportsCommand` here.

Reply:
@jimmylai, so the line `*GatherCommentsVisitor.METADATA_DEPENDENCIES,` should be removed, right?

Reply:
Yes.